requirements-dev.txt

```diff
@@ -0,0 +1,5 @@
+Django==3.0.4
```

Comment:
Any reason these don't use the [compatible release operator](https://www.python.org/dev/peps/pep-0440/#version-specifiers) `~=`?

Reply:
The compatible release operator would be great here. No reason not to use it, so I'm going to fix that, thanks!